.deployment/k8s-barista-builder/Dockerfile

```diff
@@ -27,11 +38,26 @@ RUN  curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash - \
   && apt-get update \
   && apt-get install -y libgtk-3-0 libasound2 libxss1 nodejs pv \
   && apt-get clean \
-  && npm install -g n now \
+  && npm install -g n \
   && n ${NODE_VERSION} \
   && npm -v \
   && node -v
 
 WORKDIR /home/dynatrace
 
-COPY --from=workspace /dynatrace/ ./
+
+#------------------------------------------------------------
+# The base image for barista public agent
+#------------------------------------------------------------
+FROM jenkins-agent as public
+
+RUN npm i -g vercel
+
+COPY --from=workspace-public /dynatrace/ ./
+
+#------------------------------------------------------------
+# The base image for barista internal agent
+#------------------------------------------------------------
+FROM jenkins-agent as internal
```

Comment:
cool stuff - didn't know you could branch out into multiple targets 🔥

Reply:
Yea that is the Docker builder pattern instead of having multiple files you can share layers hand have everything in one file. Faster and Cleaner.